android-auto/build.gradle.kts

```diff
@@ -40,6 +40,7 @@ dependencies {
     testImplementation("org.jetbrains.kotlinx:kotlinx-coroutines-test:1.5.1")
     testImplementation("androidx.lifecycle:lifecycle-runtime-testing:2.4.0-rc01")
     testImplementation("androidx.arch.core:core-testing:2.1.0")
+    testImplementation("org.jetbrains.kotlin:kotlin-reflect:1.5.21")
```

Comment:
Why do we need reflect here?

Reply:
It is used in `BuilderTest`. It was not compiling because of the missing dependency. 
We didn't notice that because unit tests are not run in this repository. 
Probably kotlin-reflect was a transitive dependency once.

Reply:
I see, yea, if the test was copied from the Nav SDK repo, it might be missing some deps :+1: 

:ship: